node/pkg/governor/governor.go

```diff
@@ -230,6 +306,26 @@ func (gov *ChainGovernor) initConfig() error {
 		}
 	}
 
+	for _, flowCancelConfigEntry := range flowCancelTokens {
+		addr, err := vaa.StringToAddress(flowCancelConfigEntry.addr)
+		if err != nil {
+			return err
+		}
+		key := tokenKey{chain: vaa.ChainID(flowCancelConfigEntry.chain), addr: addr}
+
+		// Only add flow cancelling for tokens that are already configured for rate-limiting.
+		if _, ok := gov.tokens[key]; ok {
+			gov.tokens[key].flowCancels = true
+		} else {
+			gov.logger.Debug("token present in flow cancel list but absent from main token list:",
+				zap.Stringer("chain", key.chain),
+				zap.Stringer("addr", key.addr),
+				zap.String("symbol", flowCancelConfigEntry.symbol),
+				zap.String("coinGeckoId", flowCancelConfigEntry.coinGeckoId),
+			)
+		}
+	}
```

Comment:
Ah I suggested we add this error, but I actually agree with Jeff here that it would be better to assert this at CI time since that's what we do with the token list already. No harm in keeping this logging in, but having a test to assert validity would be great if it's not too painful

Reply:
This will probably happen in devnet and testnet (although we don't usually have governor enabled in those cases) since we only have a single list of flow cancelled tokens, not per-environment.